engine/swarm/key-concepts.md

```diff
@@ -10,19 +10,36 @@ orchestration features of Docker Engine 1.12.
 ## What is a swarm?
 
 The cluster management and orchestration features embedded in the Docker Engine
-are built using **SwarmKit**. Docker engines participating in a cluster are
-running in **swarm mode**. You enable swarm mode for an engine by either
-initializing a swarm or joining an existing swarm.
-
-A **swarm** is a cluster of Docker engines, or _nodes_, where you deploy
-[services](key-concepts.md#services-and-tasks). The Docker Engine CLI and API
-include commands to manage swarm nodes (e.g., add or remove nodes), and deploy
-and orchestrate services across the swarm.
-
-When you run Docker without using swarm mode, you execute container
-commands. When you run the Docker in swarm mode, you orchestrate services. You can run swarm services and standalone containers on the same Docker instances.
-
-## What is a node?
+are built using [swarmkit](https://github.com/docker/swarmkit/). Swarmkit is a
+separate project which implements Dockr's orchestration layer and is used
+directly within Docker.
+
+A swarm consists of multiple Docker hosts which run in **swarm mode** and act as
+managers (to manage membership and delegation) and workers, which run
+[swarm services](key-concepts.md#services-and-tasks). When you create a service,
+you define its optimal state (number of replicas, network and storage resources
+available to it, ports the service exposes to the outside world, and more).
+Docker works to maintain that optimal state. For instance, if a worker node
+becomes unavailable, Docker schedules that node's tasks on other nodes.
+
+One of the key advantages of swarm services over standalone containers is that
+you can modify a service's configuration, including the networks and volumes
+it is connected to, without the need to restart the service. Docker will
+update the configuration and recreate the service tasks for you, in a
+configurable and managed way.
+
+When Docker is running in swarm mode, you can still run standalone containers
+on any of the Docker hosts participating in the swarm, as well as swarm
+services.
```

Comment:
Here perhaps it's important to make clear that services can only be created by sending requests to manager nodes. Containers on a given node can be created by sending requests to the Docker API at any node.